SeaPublicWebsite/Services/RecommendationService.cs

```diff
@@ -300,24 +300,61 @@ private static BrePropertyType GetBrePropertyType(PropertyType propertyType)
             }
         }
 
-        private static string GetBreConstructionDate(int? yearBuilt)
+        private static string GetBreConstructionDate(YearBuilt? yearBuilt, WallConstruction? wallConstruction, CavityWallsInsulated? cavityWallsInsulated,  HomeAge? epcConstructionAgeBand)
```

Comment:
As discussed the logic here will probably need to be changed a bit

Reply:
changed to prioritise epc data over wall insulation assumptions if it exists, there are some designs incoming to potentially just use epc data if it exists for construction data and property type (and show a confirmation of this extracted data to the user rather than these question) but I think we should merge this PR first and handle that separately